client/apps/edit/components/admin/test/components/tags.test.js

```diff
@@ -1,6 +1,6 @@
 import { mount } from 'enzyme'
 import React from 'react'
-import Article from '../../../../../../models/article'
+import Article from '../../../../../../models/article.coffee'
```

Comment:
Ah while you're in these js ones, might be nice to make them absolute? would make searching for these much easier in the future.

Reply:
But I'm also curious now - were these just not working?

Reply:
Coffee absolute paths don't work in tests because of how compiling is set up.

Reply:
also most of these references will not exist after we merge the rest of the redux things